### ISSUE TITLE
Unity Level Viewer error message(s) and question

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#338]
I'm getting some rendering issues in the HAB_Airport map.   This is right after the 0.12 update so I checked GitKraken for updates 
and it showed CathodeLib to be out of date.   After updating CathodeLib on staging I now see error messages that did not appear before.  I am going on the assumption  this is something wrong in the files for hab_airport not a bug.  But In game I see no 
problems with Mission 2, 11 or 16.  So it's only showing them here in Unity.

The reason I think it's something local is because my edited files on Steam produce the issue an unaltered version on EGS does not.
I've already tried substituting default ENV>PRODUCTION>HAB_AIRPORT>RENDERABLE with the known working defaults and this does not clear up.  So where have I gone wrong? Or is this a bug?

===
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
System.Collections.Generic.List`1[T].get_Item (System.Int32 index) (at <787acc3c9a4c471ba7d971300105af24>:0)
AlienLevelLoader.ParseComposite (CATHODE.Scripting.Composite composite, UnityEngine.GameObject parentGO, UnityEngine.Vector3 parentPos, UnityEngine.Quaternion parentRot, System.Collections.Generic.List`1[T] aliases) (at Assets/Scripts/AlienLevelLoader.cs:217)
AlienLevelLoader.ParseComposite (CATHODE.Scripting.Composite composite, UnityEngine.GameObject parentGO, UnityEngine.Vector3 parentPos, UnityEngine.Quaternion parentRot, System.Collections.Generic.List`1[T] aliases) (at Assets/Scripts/AlienLevelLoader.cs:188)
AlienLevelLoader.ParseComposite (CATHODE.Scripting.Composite composite, UnityEngine.GameObject parentGO, UnityEngine.Vector3 parentPos, UnityEngine.Quaternion parentRot, System.Collections.Generic.List`1[T] aliases) (at Assets/Scripts/AlienLevelLoader.cs:188)
AlienLevelLoader.ParseComposite (CATHODE.Scripting.Composite composite, UnityEngine.GameObject parentGO, UnityEngine.Vector3 parentPos, UnityEngine.Quaternion parentRot, System.Collections.Generic.List`1[T] aliases) (at Assets/Scripts/AlienLevelLoader.cs:188)
AlienLevelLoader.LoadCommands (CATHODE.Scripting.Composite composite) (at Assets/Scripts/AlienLevelLoader.cs:148)
AlienLevelLoader.LoadComposite (CATHODE.Scripting.ShortGuid guid) (at Assets/Scripts/AlienLevelLoader.cs:113)
WebsocketClient.Update () (at Assets/Scripts/WebsocketClient.cs:53)

**Comment by ghramsey** [comment]
After using vanilla game files for everything except the commands.pak the issue is still there.
I'm thinking this is confirming something wrong with the commands.pak itself.   
Does that error message help figure out what?

**Comment by ghramsey** [comment]
I think I just need to recreate my edits from scratch and call this done.  It's not a unity bug or an issue outside of my local files.  
I've checked backup commands.pak files back through October and all exhibit this issue.  The one from September does not.    
 So this issue is stemming from some edit clear back in probably October.  
 
 Since it's probably nothing you can help with I'll just close this.

**Comment by MattFiler** [comment]
If you've self-updated the CathodeLib submodule then you might encounter issues - are you using [the correct commit](https://github.com/OpenCAGE/CathodeLib/tree/89ec572396339512ea99e4f6b9c6dc20feb021bc)?

**Comment by ghramsey** [comment]
I actually used undo and reverted that worked.    I was thinking the graphical issues I was seeing were related to your recents upgrades to cathodelib so that was why I tried updating.  But turns out I think they're  just a corrupted commands.pak.   
![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/7c1ae795-eb88-49ac-b238-95cb91f613db)

Updating complicated the issue with those compile errors, but I'm getting a host of errors from only this one file.  
I put in a vanilla commands pak to the folder and the issues disappear.  Only using this one commands.pal causes 
the corruption in the screenshot and errors.  The entirety of the level textures for a whole section of the map is missing.
Not happening in game only here in  Unity.

This is my end though. Unless you tell me otherwise this is not a bug so much as some file corruption on this file.
